src/ontology/uberon-edit.obo

```diff
@@ -101770,7 +101771,6 @@ xref: MA:0000902
 xref: NLXANAT:090812
 xref: PBA:294021896
 is_a: UBERON:0002301 ! layer of neocortex
-relationship: develops_from UBERON:0005343 {source="BRAINSPAN"} ! cortical plate
```

Comment:
would it not be appropriate to weaken to has-developmental-contribution-from?

Reply:
oh did not know that existed, yes, definitely would be good to add that - will do